x-pack/plugins/snapshot_restore/public/application/components/restore_snapshot_form/steps/step_review.tsx

```diff
@@ -166,16 +167,19 @@ export const RestoreSnapshotStepReview: React.FunctionComponent<StepProps> = ({
                   />
                 </EuiDescriptionListTitle>
                 <EuiDescriptionListDescription>
-                  {includeGlobalState ? (
-                    <FormattedMessage
-                      id="xpack.snapshotRestore.restoreForm.stepReview.summaryTab.includeGlobalStateTrueValue"
-                      defaultMessage="Yes"
-                    />
-                  ) : (
+                  {includeGlobalState === false ? (
                     <FormattedMessage
                       id="xpack.snapshotRestore.restoreForm.stepReview.summaryTab.includeGlobalStateFalseValue"
-                      defaultMessage="No"
+                      defaultMessage="No, without any feature states"
```

Comment:
Is this extra text necessary? Aren't the feature states a separate entry?

Reply:
Since they are tightly coupled in the UI we also thought that it made sense to keep them together in the review step